pkg/pool-manager/pool_test.go

```diff
@@ -17,6 +17,8 @@ limitations under the License.
 package pool_manager
 
 import (
+	multus "github.com/intel/multus-cni/types"
+	"github.com/k8snetworkplumbingwg/kubemacpool/build/_output/bin/go/src/encoding/json"
```

Comment:
We should not import generated code from "_output". That directory is not a part of the distribution. Your IDE probably automatically loaded a wrong lib.

Reply:
yep, It was the IDE 
fixed